spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDLKeras.scala

```diff
@@ -56,11 +57,34 @@ class PythonBigDLKeras[T: ClassTag](implicit ev: TensorNumeric[T]) extends Pytho
     }
   }
 
+  def toScalaArray(list: JList[Int]): Array[Int] = {
+    if (list == null) {
+      null
+    } else {
+      list.asScala.toArray
+    }
+  }
+
   def createKerasInputLayer(
     inputShape: JList[Int] = null): Input[T] = {
     InputLayer(inputShape = toScalaShape(inputShape))
   }
 
+  def getOutputShape(module: Container[Activity, Activity, T]): JList[Int] = {
+    module.getOutputShape().toSingle().toList.asJava
+  }
+
+  def getInputShape(module: Container[Activity, Activity, T]): JList[JList[Int]] = {
+    val input = module.getInputShape()
+    val inputs = if (input.isInstanceOf[SingleShape]) {
+      MultiShape(List(input))
+    }
+    else {
+      input
+    }
+    inputs.toMulti().map(single => single.toSingle().toList.asJava).toList.asJava
```

Comment:
From the definition, nested `MultiShape` is possible, but simply process like this is OK for now. Let's add a TODO here.